bin/unit_test_script/color.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+
+# pylint template.py
+# count lines of code with:
+# sed -n '/^#cloc-start$/,/^#cloc-end$/p' template.py | cloc - --stdin-name=template.py
+
+#cloc-start
+
+# template modified 2021-09-02
+
+""" template script for future scripts """
+
+#mark - Imports
+
+from enum import Enum   #Creating Enums
+#import argparse         #command line parsing
+#import code             #interactive shell
+#import os               #file handling
+#import re               #filter in man()
+#import readline         #interactive shell
+#import subprocess       #calling unix commands
+#import sys              #exiting
```

Comment:
remove commented out imports if not needed

Reply:
that is indeed legacy code